app/models/staged/petition_creator/has_creator_signature.rb

```diff
@@ -0,0 +1,30 @@
+module Staged
+  module PetitionCreator
+    module HasCreatorSignature
+      extend ActiveSupport::Concern
+
+      included do
+        def valid?
+          super && creator_signature_valid?
```

Comment:
Argh!  So it does.  Good spot!  

We have the same problem in ``Staged::PetitionCreator::Sponsors#valid?`` too, which is weird because I went to great pains to avoid the same short-circuiting problem in ``Staged::PetitionCreator::Sponsors#sponsors_valid?``!